webapp/components/elements/FormInput.tsx

```diff
@@ -64,26 +64,34 @@ const ReceiptInput: React.FC<Props> = ({
       };
 
       return multiLine ? (
-        <Textarea
-          onBlur={(event) =>
-            props.input.onBlur(
-              event as React.FocusEvent<HTMLInputElement, Element>
-            )
-          }
-          minRows={2}
-          {...sharedProps}
-        />
+        <div>
```

Comment:
Would be nicer to not have `div` here, as it makes the description shrink and is not needed
```suggestion
        <>
```